Jellyfin.Plugin.OpenSubtitles/Jellyfin.Plugin.OpenSubtitles.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="Jellyfin.Common" Version="10.*-*" />
     <PackageReference Include="Jellyfin.Controller" Version="10.*-*" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="5.0.0" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.1.4" />
```

Comment:
Should add `<FrameworkReference Include="Microsoft.AspNetCore.App" />` instead of adding swashbuckle as a reference.

ref: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/target-aspnetcore

Reply:
Replaced in 1acf1b2db3f1f6531760dd7aa19b7b09bb2f3fa7